src/fire.js

```diff
@@ -2,12 +2,12 @@ import firebase from 'firebase';
 
   // Initialize Firebase
     var config = {
-        apiKey: "AIzaSyAf0MXQUaoblcKcy7B9zdkGfZmJsr1SqlY",
-        authDomain: "locale-dev.firebaseapp.com",
-        databaseURL: "https://locale-dev.firebaseio.com",
-        projectId: "locale-dev",
-        storageBucket: "locale-dev.appspot.com",
```

Comment:
@ivanfex  this should have been a conflict when you made a git pull.  Just wondering why you're deciding to use the testproject.firebase.